package NAlice;

option go_package = "a.yandex-team.ru/alice/begemot/lib/polyglot_merge_response/proto";
option java_package = "ru.yandex.alice.alice.begemot.lib.polyglot_merge_response.proto";

message TAliceResponsePolyglotMergerConfig {

    message TAliceParsedFramesConfig {
        enum EMergeMode {
            Default = 0;
            NativeOnly = 1;
            TranslatedOnly = 2;
            NativeOrTranslated = 3;
            TranslatedOrNative = 4;
            NativeOnlyIfExists = 5;
        }

        optional string Name = 1;
        optional EMergeMode MergeMode = 2;
    }

    optional string Language = 1;
    repeated TAliceParsedFramesConfig AliceParsedFrames = 2;
}
